app/Providers/AppServiceProvider.php

```diff
@@ -21,6 +21,9 @@ public function register(): void
             $clientId = substr(str_shuffle(MD5(microtime())), 0, $maxClientIdLength);
             $client = new Client(env('MQTT_SERVER'), env('MQTT_PORT'), $clientId);
             $client->setAuthDetails(env('MQTT_USER'), env('MQTT_PASSWORD'));
+            if (env('MQTT_TLS', false)) {
+                $client->setCryptoProtocol("tls");
```

Comment:
Please use single quoted strings since there is not variable interpolation going on here.